src/Table/Table.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useEffect } from 'react';
+import { useTable, useSortBy, useExpanded, TableState } from 'react-table';
+import { ReactComponent as ArrowUp } from './assets/arrow_drop_up.svg';
+import { ReactComponent as ArrowDown } from './assets/arrow_drop_down.svg';
+import { TableStyle } from './tableStyles';
+
+interface SortByOptions {
+    id: string;
+    desc: boolean;
+}
+
+export interface TableProps {
+    /**
+     * This is to set the data for the table.
+     */
+    data: any;
```

Comment:
Are all of the props throughout this really `any`? Can we provide better interfaces/types here to match what we or `react-table` expect? It might make more sense for TableProps to be generic (like `TableProps<T>`) and then use `T` in place of any (as/if applicable).

Reply:
Updated the type wherever possible and Also used the generic <T> type.